### ISSUE TITLE
Asymmetry between Reference Values (3.2) section and the Appraisal Policies (3.1) section

### ISSUE THREAD

**Issue by henkbirkholz** [ietf-rats-wg/architecture#297]
While Reference Values have a strong focus on conveyance and data format, Appraisal Policies do not. Is there a specific reason for this asymmetry? Additionally, there is a statement on scope on the Reference Value that does not exist for Appraisal Policies. Does that imply something?

**Comment by mcr** [comment]
Add some additional words to Appraisal policy like:  
  https://www.ietf.org/archive/id/draft-ietf-rats-architecture-10.html#section-3.2-3

**Comment by mcr** [comment]
Agreed to also move to section 6, about appraisal/etc. combinations.
